clients/drcachesim/tests/offline-burst_replace.templatex

```diff
@@ -6,8 +6,7 @@ create dir .*
 open file .*
 open file .*
 open file .*
-open file .*
```

Comment:
The number of "open file" logs == the number of threads traced, right? So now we have 3, from the previous 4?

Reply:
With lazy opening the 4th is now after the `pre-DR start` line -- but only for arm since align_endpoints is still off for x86!  A bunch of these templates have the preprocessor off so I wasn't sure I could easily add ifdefs (or comments) so I just relaxed it to handle either order.